modules/cnv-about-nmstate.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-node-network-state.adoc
+
+[id="cnv-about-nmstate_{context}"]
+= About nmstate
+
+{CNVProductNameStart} uses `nmstate` to report on and configure the state of the node network. This makes it possible to modify network policy configuration, such as by creating a linux bridge on all nodes, by applying a single configuration manifest to the cluster.
+
+Node networking is monitored and updated with the following three objects:
+
+`NodeNetworkState`:: reports the state of the network on that node.
+`NodeNetworkConfigurationPolicy`:: describes the desired network configuration on nodes. You update the node network configuration, including adding and removing interfaces from nodes, by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster. 
```

Comment:
"from nodes" is redundant, you already mention that it is about nodes in the beginning of the sentence